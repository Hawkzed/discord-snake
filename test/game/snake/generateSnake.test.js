/* eslint-env jest */
const { generateSnake } = require('../../../lib/game/snake');

describe('generateSnake', () => {
  it('generates the correct snake coordinates', () => {
    const snake = generateSnake(20, 20);

    expect(snake.length).toBe(4);
    expect(snake).toMatchSnapshot();
  });
});
