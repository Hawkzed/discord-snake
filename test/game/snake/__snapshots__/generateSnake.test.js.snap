// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generateSnake generates the correct snake coordinates 1`] = `
Array [
  Object {
    "x": 12,
    "y": 10,
  },
  Object {
    "x": 11,
    "y": 10,
  },
  Object {
    "x": 10,
    "y": 10,
  },
  Object {
    "x": 9,
    "y": 10,
  },
]
`;
